Releasing.md

```diff
@@ -14,8 +14,8 @@ This is useful in case you want to use an unreleased version of the bundle in th
 
 Currently, we are experimenting with syncing the Gutenberg Mobile release with the main app release schedule (p9ugOq-1LE-p2). We are also experimenting with performing manual tests every week (p9ugOq-1MA-p2), with a round of "smoke" test before cutting the release and a full round of tests after the release has been integrated into the main apps. A typical Gutenberg Mobile release schedule might look like the following:
```

Comment:
This is no longer something we're experimenting with, so I would drop this paragraph, and replace it with something like: "The release checklist contains the details for the release process, but here is a high level timeline..." Alternatively, we could drop the timeline here and instead just refer people to the release checklist so that there is really a single source of truth.